src/tooltip.js

```diff
@@ -233,6 +246,7 @@ angular.module('ts.utils')
           if($scope.tooltipScope)
             $scope.tooltipScope.$destroy();
           angular.element(arrowBoxContainer).remove();
+          angular.element($window).off('resize', positionTooltip);
```

Comment:
This will be the correct spot. Just make sure that you use positionTooltip direct in the `.on` also because it basically does an equality comparison to find which one to remove. So the func in `off` has to === the func in `on`